### PR TITLE
[DOR-242][AO] Add SLA message to confirmation page

### PR DESCRIPTION
**Pull request by arturopala**

### CODE REVIEW

File: app/uk/gov/hmrc/traderservices/models/CaseSLA.scala [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  def calculateFrom(sumbissionDateTime: LocalDateTime, questionsAnswers: QuestionsAnswers): CaseSLA = {
+    val slaDateTime: Option[LocalDateTime] = questionsAnswers match {
+      case answers: ExportQuestions =>
+        answers.routeType.flatMap {
+          case ExportRouteType.Hold => None
+          case _ =>
+            answers.freightType.map {
+              case ExportFreightType.Air      => sumbissionDateTime.plusHours(2)
+              case ExportFreightType.RORO     => sumbissionDateTime.plusHours(2)
+              case ExportFreightType.Maritime => sumbissionDateTime.plusHours(3)
```
Comment (Tasmine10ten) [inline app/uk/gov/hmrc/traderservices/models/CaseSLA.scala:50]: Maritime exports are 2hrs

File: app/uk/gov/hmrc/traderservices/models/CaseSLA.scala [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      case answers: ImportQuestions =>
+        answers.routeType.flatMap {
+          case ImportRouteType.Hold => None
+          case _ =>
+            answers.freightType.map {
+              case ImportFreightType.Air  => sumbissionDateTime.plusHours(2)
+              case ImportFreightType.RORO => sumbissionDateTime.plusHours(2)
+              case ImportFreightType.Maritime =>
+                if (sumbissionDateTime.isAfter(sumbissionDateTime.withHour(15).withMinute(0).withSecond(0).withNano(0)))
+                  sumbissionDateTime.plusDays(1).withHour(8).withMinute(0).withSecond(0).withNano(0)
```
Comment (Tasmine10ten) [inline app/uk/gov/hmrc/traderservices/models/CaseSLA.scala:62]: Works well overall but after midnight it uses the normal logic.

ie. 
Submit at 23.59 => 8.00
Submit at 00.00 => 3.00 

I double checked with Meg and we want to run this up until 5am so it has the standard 3hrs, so if a case goes in at: 
1500 - 0459 => 8.00
0500+ => return to standard logic (+3 hrs)

File: test/uk/gov/hmrc/traderservices/models/CaseSLASpec.scala [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+        for (routeType <- ExportRouteType.values.filterNot(_ == ExportRouteType.Hold))
+          s"calculate three hours SLA for export case if route type is $routeType, and freightType is Maritime" in {
+            CaseSLA.calculateFrom(
+              sumbissionDateTime,
+              ExportQuestions(
+                routeType = Some(routeType),
+                freightType = Some(ExportFreightType.Maritime)
+              )
+            ) shouldBe CaseSLA(Some(sumbissionDateTime.plusHours(3)))
```
Comment (Tasmine10ten) [inline test/uk/gov/hmrc/traderservices/models/CaseSLASpec.scala:138]: 2hrs (rest all look good)

File: app/uk/gov/hmrc/traderservices/models/CaseSLA.scala (hunk 2 above)
Comment (arturopala) [reply app/uk/gov/hmrc/traderservices/models/CaseSLA.scala:62]: ok, but this is new logic criteria

File: app/uk/gov/hmrc/traderservices/models/CaseSLA.scala (hunk 1 above)
Comment (arturopala) [reply app/uk/gov/hmrc/traderservices/models/CaseSLA.scala]: fixed

File: test/uk/gov/hmrc/traderservices/models/CaseSLASpec.scala (hunk 3 above)
Comment (arturopala) [reply test/uk/gov/hmrc/traderservices/models/CaseSLASpec.scala]: fixed

File: app/uk/gov/hmrc/traderservices/models/CaseSLA.scala (hunk 2 above)
Comment (Tasmine10ten) [reply app/uk/gov/hmrc/traderservices/models/CaseSLA.scala:62]: Yeah again not detailed in story but I will add - thank you

**Pull request merged**